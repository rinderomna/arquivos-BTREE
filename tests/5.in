6 tipo1 binario5.bin indice5.bin 3
1 id 555
2 id 243 sigla "MG"
1 id 870
