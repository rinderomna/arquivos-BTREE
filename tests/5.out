24515.840000
14591.970000
